test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -447,5 +447,34 @@ public void AddProductIncorrectCasing(SupportedLanguages lang)
             Assert.Throws<AggregateException>(() => this.SendOutputGetRequest("addproduct-incorrectcasing").Wait());
             Assert.Equal(0, this.ExecuteScalar("SELECT COUNT(*) FROM Products"));
         }
+
+        /// <summary>
+        /// Tests that subsequent upserts work correctly when the object properties are different from the first upsert.
+        ///
+        [Theory]
+        [SqlInlineData()]
+        public void AddProductWithDifferentPropertiesTest(SupportedLanguages lang)
```

Comment:
AddProductWithDifferentPropertiesTest is confusing, AddConsecutiveProductsTest ?

Reply:
It's not just about adding consecutive products though (we do that in other tests). It's specifically testing that calling the function with different sets of properties still results in the row being updated. So I think the name is fine (although if you can think of a better one Lucy go ahead)

Reply:
At first look, I was looking for different properties and was confused and then read the test comments and understood. It's just a nit, we can move on if you can't come up with a better name :)